Doc/whatsnew/3.14.rst

```diff
@@ -561,6 +561,9 @@ os
   to the :mod:`os` module.
   (Contributed by James Roy in :gh:`127688`.)
 
+* Add the :func:`os.readinto` function to read into a
+  :ref:`buffer object <bufferobjects>` from a file descriptor.
+  (Contributed by Cody Maloney in :gh:`129205`.)
```

Comment:
```suggestion
  (Contributed by Cody Maloney in :gh:`129205`.)

```

Just to be consistent with the other sections (we'll anyway do a copy-edit before the release I think)